torch/utils/_device.py

```diff
@@ -0,0 +1,62 @@
+import torch
+from torch.overrides import TorchFunctionMode
+import functools
+
+@functools.lru_cache(1)
+def _device_constructors():
+    return {
+        # standard ones
+        torch.empty,
+        torch.empty_strided,
+        torch.empty_quantized,
+        torch.ones,
+        torch.arange,
+        torch.bartlett_window,
+        torch.blackman_window,
+        torch.eye,
+        torch.fft.fftfreq,
+        torch.fft.rfftfreq,
+        torch.full,
+        torch.fill,
+        torch.hamming_window,
+        torch.hann_window,
+        torch.kaiser_window,
+        torch.linspace,
+        torch.logspace,
+        torch.nested.nested_tensor,
+        # torch.normal,
```

Comment:
Why?

Reply:
This constructor, for some godforsaken reason, doesn't actually take a device argument 🤣